src/pages/NewChairConsole.js

```diff
@@ -1,40 +1,426 @@
+      width: "10%",
+      margin: "0 2%",
+      display: "flex",
+      justifyContent: "left",
+      alignItems: "center",
+    },
+    DAC_CELL: {
+      width: "10%",
+      margin: "0 2%",
+      display: "flex",
+      justifyContent: "left",
+      alignItems: "center",
+    },
+    ELECTION_STATUS_CELL: {
+      width: "10%",
+      margin: "0 2%",
+      display: "flex",
+      justifyContent: "left",
+      alignItems: "center",
+    },
+    ELECTION_ACTIONS_CELL: {
+      width: "23%",
+      margin: "0 2%",
+      display: "flex",
+      justifyContent: "left",
+      alignItems: "center",
+    }
+  },
+  MODAL: {
+    CONTENT: {
+      height: "auto",
+      top: "20%",
+      bottom: "20%",
+      left: "20%",
+      right: "20%",
+      fontFamily: 'Montserrat',
+      padding: "2%"
+    },
+    DAR_SUBHEADER: {
+      display: 'flex',
       fontFamily: 'Montserrat',
+      fontSize: '16px',
+      fontWeight: Theme.font.weight.semibold,
+      justifyContent: 'left',
+      color: "#777"
+    },
+    TITLE_HEADER: {
+      display: 'flex',
+      fontFamily: 'Montserrat',
+      fontSize: Theme.font.size.title,
+      fontWeight: Theme.font.weight.regular,
+      justifyContent: 'left',
+      marginBottom: '4%'
+    },
+    DAR_DETAIL_ROW: {
+      padding: '0 3%',
+      margin: '2% 0',
+      display: 'flex',
+      justifyContent: 'space-between'
+    },
+    DAR_LABEL: {
+      color: "#777777",
       fontSize: '14px',
-      fontStyle: 'normal'
+      fontWeight: Theme.font.weight.semibold,
+      width: "25%",
+      textAlign: 'right'
+    },
+    DAR_DETAIL: {
+      fontSize: '16px',
+      fontWeight: Theme.font.weight.medium,
+      width: "70%"
+    }
+  }
+};
+
+const getDatasetNames = async(datasetIds) => {
+  let datasetNames = [];
+  if(!isEmpty(datasetIds)) {
+    try{
+      const datasetPromises = datasetIds.map(datasetId => {
+        return DataSet.getDataSetsByDatasetId(datasetId);
+      });
+      const datasets = await Promise.all(datasetPromises);
+      datasetNames = datasets.map((dataset) => {
+        let nameProperty = find((property) => {
+          return property.propertyName === "Dataset Name";
+        }, dataset.properties);
+        return nameProperty.propertyValue;
+      });
+    } catch(error) {
+      Notifications.showError({text: 'Error: Failed to initilize datasets for modal'});
+    }
+  }
+  return datasetNames.join('\n');
+};
+
+const getResearcherProperties = async(userId) => {
+  if(!isNil(userId)) {
+    let researcherProperties = {};
+    try{
+      const researcherInfo = await User.getById(userId);
+      researcherInfo.researcherProperties.forEach((property) => {
+        researcherProperties[property.propertyKey] = property.propertyValue;
+      });
+      return Object.assign({}, researcherInfo, researcherProperties);
+    }catch(error){
+      Notifications.showError({text: 'Error: Failed to initialize researcher information'});
+      return {};
+    }
+  }
+};
```

Comment:
This, along with `getDatasetNames` are modal specific functions that I think would be better off if we pulled them out into a separate chair console summary modal component. It would also provide an opportunity to extract out all of the modal style info which would clean up this class a lot.